modules.json

```diff
@@ -0,0 +1,16 @@
+[
```

Comment:
Should probably be unquoted properties, and single quoted strings

https://github.com/airbnb/javascript#objects--quoted-props